src/components/layout/index.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import AccountPopup from './account-popup';
+import './_.css';
+
+type THeader = {
+  is_login?: boolean;
+  currency?: any;
+  accounts?: any;
+};
+
+const Header = ({
+  accounts = {
+    Demo: 'VRTC124124',
+    Real: 'CR214124',
+    icon: 'public/images/demo-currency.svg',
+  },
+  currency = { Demo: 10000, Real: 100 },
+}: THeader) => {
+  const [is_open, setOpen] = useState(false);
+  const [is_hover, setHover] = useState(false);
+  const [is_login, setLogin] = useState(false);
+
+  const onRequestClose = () => {
+    setOpen(false);
+  };
+
+  return (
+    <>
+      <div className="header-menu">
+        <div className="header-menu-left">
+          <div className="header-menu-platform">DTrader Air</div>
+          <a
```

Comment:
@NikitK-deriv i suggest mapping links based on their data, same for all repeatedly used code blocks/tags 🙂

Reply:
Done @maryia-binary please check 🙏